views/index.erb

```diff
@@ -1,14 +1,35 @@
 <% if @profile %>
-  <h3><%= @profile["first_name"] %>'s Amazing Profile</h3>
+
+  <div id="picNheader">
+    <div id="profilePic" >
+      <img src='https://graph.facebook.com/37109651/picture?width=200&height=200' />
+    </div>
+
+    <h3 id="profileHeader"><%= @profile["first_name"] %>'s Amazing Profile<br><%= @profile["about"] %></h3>
+  </div>
 
   <!-- TODO- Perhaps re-jig this to fit your new design / layout -->
+
   <% @profile.each do |field, value| %>
-    <div class="profile-section box">
-      <h4><%= field %></h4>
-      <p class="profile-value"><%= value %></p>
-    </div>
+    <% unless field == "about" %>
+      <div class="profile-section box">
+        <% if field == "dob" %>
+          <h4>Date of Birth</h4>
+          <p class="profile-value"><%= Time.parse(value).strftime("%d %B %Y") %></p>
+        <% else %>
+        <h4><%= field.capitalize.gsub("_", " ").split.map(&:capitalize).join(' ') %></h4>
```

Comment:
There is some inefficiency here in how you're calling `capitalize` twice. How could we slim this code down and make it more concise?